src/components/ChainLogo.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import styled from 'styled-components';
+import ArbitrumBadge from '../assets/badges/arbitrumBadge.png';
+import ArbitrumBadgeDark from '../assets/badges/arbitrumBadgeDark.png';
+import ArbitrumBadgeNoShadow from '../assets/badges/arbitrumBadgeNoShadow.png';
+import OptimismBadge from '../assets/badges/optimismBadge.png';
+import OptimismBadgeDark from '../assets/badges/optimismBadgeDark.png';
+import OptimismBadgeNoShadow from '../assets/badges/optimismBadgeNoShadow.png';
+import PolygonBadge from '../assets/badges/polygonBadge.png';
+import PolygonBadgeDark from '../assets/badges/polygonBadgeDark.png';
+import PolygonBadgeNoShadow from '../assets/badges/polygonBadgeNoShadow.png';
+import { ImgixImage } from './images';
+import { Centered } from './layout';
+import networkTypes from '@rainbow-me/helpers/networkTypes';
+import { position } from '@rainbow-me/styles';
+
+const ChainIcon = styled(ImgixImage)`
```

Comment:
we talked about FastImage, with `ImgixImage` I had a weird behavior changing network

Follow
1. switch Mainnet => Optimism: The image is correctly updated to Optimism
2. switch Optimism => arbitrum: The image is still Optimism logo
3. switch arbitrum => polygon: The image is the Arbitrum logo